src/pkg/packager/create.go

```diff
@@ -550,6 +552,62 @@ func (p *Packager) addComponent(index int, component types.ZarfComponent, isSkel
 		}
 	}
 
+	if len(component.Reports) > 0 {
+		spinner := message.NewProgressSpinner("Loading %d reports", len(component.Reports))
+		defer spinner.Success()
+
+		err = utils.CreateDirectory(componentPath.Reports, 0700)
+		if err != nil {
+			return fmt.Errorf("unable to create reports destination directory: %w", err)
+		}
+
+		for _, report := range component.Reports {
+			var filePath string
+			switch reportType := strings.ToLower(report.Type); reportType {
+			case "vex":
+				dst := fmt.Sprintf("%s/%s", componentPath.Reports, reportType)
+				err = utils.CreateDirectory(dst, 0700)
+				if err != nil {
+					return fmt.Errorf("unable to create reports destination directory: %w", err)
+				}
+				if utils.IsURL(report.Source) {
+					message.Debug("Source was identified as a URL")
+					filePath = fmt.Sprintf("%s/%s/%s", componentPath.Reports, reportType, report.Name)
```

Comment:
dont use `/` in filepaths. Use `filepath.Join` as it will use OS specific separators

Reply:
updated